app/services/optout/OptOutProposition.scala

```diff
@@ -18,6 +18,7 @@ package services.optout
 
 import models.incomeSourceDetails.TaxYear
 
+
 case class OptOutProposition(previousTaxYear: PreviousOptOutTaxYear,
```

Comment:
the method 'optOutForSingleYear' line 35 can be removed now

Reply:
done